src/test/java/edu/iu/uits/lms/etextmanager/services/ETextItLoggingTest.java

```diff
@@ -0,0 +1,151 @@
+package edu.iu.uits.lms.etextmanager.services;
```

Comment:
This fails unless one gives the -P it12 build arg as it needs that it12 lib. Not sure how to set this up so this is an optional test because even if I mark the test with @profile(it12), when one doesn't provide -P it12 the import itself fails on build

package edu.iu.es.esi.audit does not exist